### PR TITLE
Fix output file path detection

### PR DESCRIPTION
**Pull request by dmy3k**
Output path for writing headroom is incorrectly detected when case path is absolute in config. This PR fixes that.

### CODE REVIEW

**Review by Zagryzyn — APPROVED** [review]

File: gridcapacity/output.py [hunk 1]
```diff
@@ -115,9 +115,10 @@ def write_exported_data(case_name: str, exported_data: ExportedData) -> None:
 
 
 def get_output_folder(case_path: Path) -> Path:
-    return (
-        case_path if case_path.is_absolute() else Path(__file__).absolute().parents[1]
-    )
+    if case_path.is_absolute():
+        return case_path if case_path.is_dir() else case_path.parent
+    else:
+        return Path(__file__).absolute().parents[1]
```
Comment (dvlazariev) [inline gridcapacity/output.py:121]: That was an intended behavior. And how a case path could be a directory?

**Review by dvlazariev — CHANGES_REQUESTED** [review]

File: gridcapacity/output.py [hunk 2]
```diff
@@ -115,9 +115,10 @@ def write_exported_data(case_name: str, exported_data: ExportedData) -> None:
 
 
 def get_output_folder(case_path: Path) -> Path:
-    return (
-        case_path if case_path.is_absolute() else Path(__file__).absolute().parents[1]
-    )
+    if case_path.is_absolute():
+        return case_path if case_path.is_dir() else case_path.parent
```
Comment (dvlazariev) [inline gridcapacity/output.py:119]:
```suggestion
        return case_path.parent
```

**Review by dvlazariev — APPROVED** [review]

File: gridcapacity/output.py (hunk 1 above)
Comment (dvlazariev) [reply gridcapacity/output.py:121]: OK, there was a bug.

File: gridcapacity/output.py (hunk 2 above)
Comment (dmy3k) [inline gridcapacity/output.py:119]: Agreed to simplify a bit

```suggestion
        return case_path.parent
```

**Pull request merged**